<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>882</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>882</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="funcion_line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>781</width>
      <height>45</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_enter">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ravno">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pre_line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>44</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">button_group_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_ac">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>56</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_unar">
    <property name="geometry">
     <rect>
      <x>606</x>
      <y>130</y>
      <width>56</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>unar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mod">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>130</y>
      <width>56</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_minus">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>250</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mull">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_div">
    <property name="geometry">
     <rect>
      <x>718</x>
      <y>130</y>
      <width>56</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dot">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_x">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="line_x">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_log">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sqrt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tan">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_acos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_atan">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ln">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_asin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="line_max_x">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>107</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="prefix">
     <string>max x: </string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999996.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="line_min_x">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>136</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="prefix">
     <string>min x:  </string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_draw_graphic">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>draw
graphic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open_scobka">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>51</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>200</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close_scobka">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>-10</y>
      <width>51</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>200</pointsize>
     </font>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_backspace">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>881</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;-----------------------------------------------------------------------------------------------------</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graphic" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>501</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exp">
    <property name="geometry">
     <rect>
      <x>774</x>
      <y>130</y>
      <width>56</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_deposit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>343</y>
      <width>51</width>
      <height>96</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deposit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_credit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>343</y>
      <width>51</width>
      <height>95</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLenya_style">
    <property name="title">
     <string>Lenya style</string>
    </property>
   </widget>
   <addaction name="menuLenya_style"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="button_group_numbers"/>
 </buttongroups>
</ui>
